src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingStatementTests.cs

```diff
@@ -477,7 +477,7 @@ .maxstack  1
     IL_0021:  ldloc.0
     IL_0022:  brfalse.s  IL_002a
     IL_0024:  ldloc.0
-    IL_0025:  callvirt   ""void System.IDisposable.Dispose()""
+    IL_0025:  callvirt   ""void DisposableClass.Dispose()""
     IL_002a:  endfinally
```

Comment:
This looks like a semantically observable change in existing code. What if we had created a subclass of `DisposableClass` where the subclass provides a different implementation of the interface method? Then we would be using a different method by this code gen strategy than previously.

Reply:
I think we may have to go back to the drawing board to understand exactly what we want this language feature implementation to do in order to avoid any change to existing code.

---
In reply to: [194901108](https://github.com/dotnet/roslyn/pull/27729#discussion_r194901108) [](ancestors = 194901108)

Reply:
(I'm stopping review here as I think we need to discuss this)

---
In reply to: [194901406](https://github.com/dotnet/roslyn/pull/27729#discussion_r194901406) [](ancestors = 194901406,194901108)